src/stretchy.js

```diff
@@ -154,98 +153,108 @@ const groupLength = function(arg: ParseNode): number {
+                "width": "100%",
+                "height": height + "em",
+                "viewBox": `0 0 ${viewBoxWidth} ${viewBoxHeight}`,
+                "preserveAspectRatio": "none",
+            });
+            return {
+                span: buildCommon.makeSpan([], [svgNode], options),
+                minWidth: 0,
+                height,
+            };
+        } else {
+            const spans = [];
 
-            attributes = [["width", "400em"], ["height", height + "em"]];
-            attributes.push(["viewBox", `0 0 ${viewBoxWidth} ${viewBoxHeight}`]);
+            const [paths, minWidth, viewBoxHeight, align1] = katexImagesData[label];
 
-            if (numSvgChildren === 2) {
-                widthClass = ["halfarrow-left", "halfarrow-right"][i];
-                align = ["xMinYMin", "xMaxYMin"][i];
-            } else  if (numSvgChildren === 3) {
-                widthClass = ["brace-left", "brace-center", "brace-right"][i];
-                align = ["xMinYMin", "xMidYMin", "xMaxYMin"][i];
+            const numSvgChildren = paths.length;
+            if (1 > numSvgChildren || numSvgChildren > 3) {
+                throw new Error(
+                    `Correct katexImagesData or update code below to support
+                    ${numSvgChildren} children.`);
             }
+            const height = viewBoxHeight / 1000;
+
+            for (let i = 0; i < numSvgChildren; i++) {
+                const path = new domTree.pathNode(paths[i]);
+
+                let align;
+                let widthClass;
+                if (numSvgChildren === 1) {
+                    widthClass = "hide-tail";
+                    align = align1;
+                } else if (numSvgChildren === 2) {
+                    widthClass = ["halfarrow-left", "halfarrow-right"][i];
+                    align = ["xMinYMin", "xMaxYMin"][i];
+                } else /* if (numSvgChildren === 3) */ {
+                    widthClass = ["brace-left", "brace-center", "brace-right"][i];
+                    align = ["xMinYMin", "xMidYMin", "xMaxYMin"][i];
+                }
+
+                const svgNode = new domTree.svgNode([path], {
+                    "width": "400em",
+                    "height": height + "em",
+                    "viewBox": `0 0 ${viewBoxWidth} ${viewBoxHeight}`,
+                    "preserveAspectRatio": align + " slice",
+                });
 
-            attributes.push(["preserveAspectRatio", align + " slice"]);
-            svgNode = new domTree.svgNode([path], attributes);
-
-            if (numSvgChildren === 1) {
-                spans.push(buildCommon.makeSpan(["hide-tail"], [svgNode], options));
-            } else {
                 const span = buildCommon.makeSpan([widthClass], [svgNode], options);
-                span.style.height = height + "em";
-                spans.push(span);
+                if (numSvgChildren === 1) {
+                    return {span, minWidth, height};
```

Comment:
You suggested this in the stretchy PR. I did this anyway to resolve some of the type errors. (It was otherwise interpreting the returned span to have type `span | svgNode`.